lava_test_plans/devices/hi6220-hikey

```diff
@@ -4,15 +4,15 @@
 {% set BOOT_OS_PROMPT = BOOT_OS_PROMPT|default("hikey:~") %}
 {% set DEPLOY_OS = DEPLOY_OS|default("oe") %}
 {% set PTABLE_LABEL = "ptable" %}
```

Comment:
This one can be dropped since its set in include/fastboot.jinja2 as ptable as default.